app/controllers/application_controller.rb

```diff
@@ -3,11 +3,17 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  def admin_authorize!
+    if current_user.role != 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.